en/02_Developer_Guides/02_Controllers/02_Routing.md

```diff
@@ -70,6 +66,22 @@ SilverStripe\Control\Director:
 
 ## Parameters
 
+> [!CAUTION]
+> Be aware that if your action doesn't follow the default URL handler pattern `$Action//$ID/$OtherID`, you *must* declare the appropriate url_handler pattern for your action.
+> This is because the `Director.rules` configuration is *only* used to indentify which *controller* should handle the request, and how to handle parameters. It does *not*
+> provide enough information on its own for the controller to know which *action* should be used.
+>
+> For example, the following two routing rules *must* have an appropriate `url_handlers` declaration:
+>
+> - `teams//$Action/$ID/$AnotherID/$Name` - the `$Action/$ID/$AnotherID/$Name` portion needs to be declared in `url_handlers`
+> - `teams//$@` - the `$@` portion needs to be declared in `url_handlers`
+>
+> In both cases, having any more than 3 path segments after `teams/` in the URL will result in the error "I can't handle sub-URLs on class `App\Control\TeamController`". This happens because there are more path segments than the default URL handler pattern knows how to deal with.
+>
+> Note also that in both cases the first path segment after `teams/` will try to match against an action on the controller. You can also use `url_handlers` to declare a specific action that should handle these patterns regardless of what the parameter values resolve to.
+>
+> See [URL Handlers](#url-handlers) below for more information about the `url_handlers` configuration array.
```

Comment:
Moved this from below to satisfy [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.